staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -394,6 +394,15 @@ func (a *GenericAdmissionWebhook) exemptedByNamespaceSelector(h *v1alpha1.Webhoo
 		return true, nil
 	}
 	namespaceLabels, err := a.getNamespaceLabels(attr)
+	// this means the namespace is not found, for backwards compatibility,
```

Comment:
Isn't there a namespace checker admission plugin? Maybe it needs to be placed in the list before webhooks?

Reply:
> Isn't there a namespace checker admission plugin? 

The checker always allow DELETE request: https://github.com/kubernetes/kubernetes/blob/1418f9845237840a3e285cad3a763244c4deaaab/staging/src/k8s.io/apiserver/pkg/admission/plugin/namespace/lifecycle/admission.go#L113

Also, we need this webhook to function correctly even if the namespace plugin is turned off.

>  Maybe it needs to be placed in the list before webhooks?

It is :)